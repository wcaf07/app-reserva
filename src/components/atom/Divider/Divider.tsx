import colors from '@tokens/colors';
import styled from 'styled-components/native';

const Divider = styled.View`
  height: 1px;
  width: 100%;
  background-color: ${colors.gray};
  margin-top: 5px;
  margin-bottom: 5px;
`;

export default Divider;
