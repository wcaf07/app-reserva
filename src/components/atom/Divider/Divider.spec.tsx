import React from 'react';
import {render} from '@testing-library/react-native';
import Divider from './Divider';

describe('Divider.styles', () => {
  test('THEN Divider SHOULD match with snapshot when is show', () => {
    const component = render(<Divider />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
