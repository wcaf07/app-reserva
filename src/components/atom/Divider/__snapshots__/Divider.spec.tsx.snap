// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider.styles THEN Divider SHOULD match with snapshot when is show 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#d8d8d8",
        "height": 1,
        "marginBottom": 5,
        "marginTop": 5,
        "width": "100%",
      },
    ]
  }
/>
`;
